pkg/util/ssh.go

```diff
@@ -142,19 +142,19 @@ func (s *SSHTunnel) Close() error {
 	return nil
 }
 
-func RunSSHCommand(cmd, host string, signer ssh.Signer) (string, string, int, error) {
+func RunSSHCommand(cmd, user, host string, signer ssh.Signer) (string, string, int, error) {
```

Comment:
typical pattern would be something like:

```
type sshDialer interface {
    Dial(...) error
}

type realDialer struct{}

func (d *realDialer) Dial() {
    // real dial
}

func RunSSHCommand(cmd, user, host string, signer ssh.Signer) (string, string, int, error) {
  return runSSHCommand(&realDialer{}, cmd, user, host, signer)
}

func runSSHCommand(dialer sshDialer, cmd, user, host string, signer ssh.Signer) (string, string, int, error) {
  // real code
}
```

Then test runSSHCommand and play ostritch with RunSSHCommand